### PR TITLE
MINOR FIX : Calendar Entry Attachment Service Exception

### PR DESCRIPTION
**Pull request by imjane5**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.Add.cs [hunk 1]
```diff
@@ -94,8 +94,11 @@ public async Task ShouldThrowServiceExceptionOnAddWhenExceptionOccursAndLogItAsy
             CalendarEntryAttachment someCalendarEntryAttachment = CreateRandomCalendarEntryAttachment();
             var serviceException = new Exception();
 
+            var failedCalendarEntryAttachment =
+                new FailedCalendarEntryAttachment(serviceException);
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.Add.cs:98]: FailedCalendarEntry**Service**Attachment

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.Add.cs [hunk 2]
```diff
@@ -95,7 +95,7 @@ public async Task ShouldThrowServiceExceptionOnAddWhenExceptionOccursAndLogItAsy
             var serviceException = new Exception();
 
             var failedCalendarEntryAttachment =
-                new FailedCalendarEntryAttachment(serviceException);
+                new FailedCalendarEntryServiceAttachment(serviceException);
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.Add.cs]: Sorry for the typo previously, it is Failed<Entity>ServiceException so it is FailedCalendarEntryAttachmentServiceException

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.Add.cs (hunk 1 above)
Comment (imjane5) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.Add.cs]: Updated. Please check

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.Add.cs [hunk 3]
```diff
@@ -94,8 +94,11 @@ public async Task ShouldThrowServiceExceptionOnAddWhenExceptionOccursAndLogItAsy
             CalendarEntryAttachment someCalendarEntryAttachment = CreateRandomCalendarEntryAttachment();
             var serviceException = new Exception();
 
+            var failedCalendarEntryAttachmentService =
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.Add.cs:97]: failedCalendarEntryAttachmentServiceException should be the variable name. Please verify in all the places.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**